core/jvm/src/main/scala/cats/effect/unsafe/metrics/ComputePoolSamplerMBean.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020-2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect.unsafe.metrics
+
+/**
+ * An MBean interface for monitoring the IO work stealing compute pool.
+ */
+trait ComputePoolSamplerMBean {
+
+  /**
+   * Returns the number of worker threads in the compute pool.
+   *
+   * @return the number of worker threads in the compute pool
+   */
+  def getWorkerThreadCount: Int
```

Comment:
Do nullary methods get compiled to normal jvm zero-arg methods with `()`?

Because the beans are a java-ish thing, we may want to enforce explicit `()` application for the methods in this interface.

```suggestion
  def getWorkerThreadCount(): Int
```

Reply:
I honestly followed your example in the linked issue 😅. I didn't give it too much thought. But it might be a good change.

Reply:
Well, I suppose if what I pasted worked, then it just works, because that's what I have in prod :)